### DISCUSSION TITLE
Get records by multiple relation

### DISCUSSION THREAD

**Discussion by ChrisPoul** [pocketbase/pocketbase#3775] [Q&A] [answered]
I have a collection `loans` and each loan has a relation with multiple `users`, I want to get all the loans related to a user by using the `getList` method. I have tried using this filter `filter: "users.id ~ {userId}"` but it gives me an error "invalid right operand \"z5kcgnifek2zu4l\" - unknown field \"z5kcgnifek2zu4l\"", I also tried using the `=` operator and `?=` but it gave me the same error.

I could just pull the user and expand it's loans, but I want to see if it's possible to do it this way.

**Comment by ganigeorgiev** [comment, marked as answer]
It should work but the string id value must be single or double quoted, otherwise it will be treated as field identifier. You can try with:

```js
const userId = "..."

const result = await pb.collection("loans").getList(1, 20, {
    // the same as `users.id ?= "${userId}"`
    filter: pb.filter("users.id ?= {:id}", { id: userId })
})
```

**Reply by ChrisPoul** [reply]
This worked perfectly, thank you for responding so fast
